sdk/src/beta9/runner/taskqueue.py

```diff
@@ -333,15 +339,25 @@ def process_tasks(self, channel: Channel) -> None:
                             if not complete_task_response.ok:
                                 raise RunnerException("Unable to end task")
 
-                            print(f"Task completed <{task.id}>, took {duration}s")
+                            if task_status != TaskStatus.Retry:
```

Comment:
This feels weird. Retry is not the common path, and you're first checking if status is retry, but only for a log. You can alternatively check for retry and if retry, just return right away instead of the else block.